### PR TITLE
Fixes #32

### PR DESCRIPTION
**Pull request by tmb**
Fix issue #32

### CODE REVIEW

**Review by lachlanjc — COMMENTED** [review]
- I don’t think return belongs here
- Should this be a form validation error instead? Not sure

**Comment by tmb** [comment]
@lachlanjc return is needed because it'll double render otherwise, not too sure about the second one.

is it cool to merge this? it's a pretty small change and is functional

File: app/controllers/organizer_position_invites_controller.rb [hunk 1]
```diff
@@ -17,6 +17,11 @@ def create
 
     authorize @invite
 
+    if event.users.pluck(:email).include? @invite.email
```
Comment (maxwofford) [inline app/controllers/organizer_position_invites_controller.rb]: @tmb this should be done with model validations

**Review by maxwofford — CHANGES_REQUESTED** [review]

File: app/controllers/organizer_position_invites_controller.rb (hunk 1 above)
Comment (zachlatta) [reply app/controllers/organizer_position_invites_controller.rb]: So, just to clarify, you can add something like the following to the invite model:

```ruby
validates :not_already_invited

private

def not_already_invited
  # add logic + errors.add(...) here
end

**Comment by tmb** [comment]
moved! sorry this took so long, ended up getting distracted implementing another feature which I eventually realized should be on the `one` branch (where I am now putting it!)

File: app/models/organizer_position_invite.rb [hunk 2]
```diff
@@ -83,4 +84,15 @@ def reject
   def rejected?
     self.rejected_at.present?
   end
+
+  private
+
+  def not_already_organizer
+    if self.event.users.pluck(:email).include? self.email
+      self.errors.add(:user, 'is already an organizer of this event!')
+      return false
```
Comment (maxwofford) [inline app/models/organizer_position_invite.rb]: Does this need a `return` here?

Comment (tmb) [reply app/models/organizer_position_invite.rb]: I think so, doesn't it signify to `:validate` that the check failed?

Comment (maxwofford) [reply app/models/organizer_position_invite.rb]: Looking at the examples in the docs, doesn't seem like they use them (https://guides.rubyonrails.org/active_record_validations.html#custom-methods)

Comment (tmb) [reply app/models/organizer_position_invite.rb]: 👍hmmm ok will change!

**Comment by tmb** [comment]
merging 😄

File: app/models/organizer_position_invite.rb (hunk 2 above)
Comment (zachlatta) [reply app/models/organizer_position_invite.rb]: > The valid? method will verify that the errors collection is empty, so your custom validation methods should add errors to it when you wish validation to fail:

**Pull request merged**